src/information-panel/information-panel.scss

```diff
@@ -76,18 +76,34 @@
     height: $iui-baseline * 6;
     display: flex;
     align-items: center;
+    justify-content: space-between;
     @include themed {
       background-color: t(iui-color-background-3);
     }
 
-    .iui-selection-icon {
-      width: $iui-l;
-      height: $iui-l;
-      margin-right: $iui-s;
+    .iui-header-label {
+      display: flex;
+      align-items: center;
+      overflow: hidden;
+
+      span {
```

Comment:
not sure we should tackle just span... Dont like, but maybe just `*` in here for handling overflow...?